### ISSUE TITLE
Bug with field of view and wait action

### ISSUE THREAD

**Issue by Devengel** [SelinaDev/Godot-Roguelike-Tutorial#4]
Hi, i found a bug with field of view. 
in game.gd
```
		if action.perform():
			if player.grid_position != previous_player_position:
				map.update_fov(player.grid_position)

```
if an enemy is in a player's potential field of view, but the player uses the wait action, it will not get shown and the player will be continously attacked until they move or kill the enemy, i don't know how to fix this, So for me i just got rid of the if statement  that checks if player moved and update the field of view regardless.

**Comment by SelinaDev** [comment]
Hi! Thank you for mentioning that! It seems that is indeed a logical error on my part. I thought that recalculating was only necessary when moving, and didn't account for the situation you described.
Removing the if statement should indeed fix that problem. I will make sure to change the code (and tutorial) accordingly.

**Comment by SelinaDev** [comment]
Should now be solved by 71e7a8e (also updated on the website). Thanks again!
